package.json

```diff
@@ -17,16 +17,22 @@
     "type": "git",
     "url": "git://github.com/xmldom/xmldom.git"
   },
-  "main": "./dom-parser.js",
+  "main": "domParser.js",
+  "files": [
```

Comment:
If we would add `files`, which I would definitely favor, then I would favor the following approach:

- move the sources into a `lib` subdirectory and keeping a single main index JavaScript file such as index.js as the main entry point.
- The main entry point should export several objects such as `dom` and `domParser`, which would likely be a breaking API change.
- Update the types in **/.d.ts files if needed

Reply:
Moved the files to `lib`. Holding off on other refactoring for now (like exports). Can handle that in another PR down the road since this one is already fairly dense as you noted.

Reply:
Also I agree we should squash the commits before merging.